com.unity.render-pipelines.high-definition/Editor/RenderPipeline/PathTracing/PathTracingEditor.cs

```diff
@@ -2,8 +2,8 @@
 
 using UnityEditor.Rendering;
 using UnityEditor.Rendering.HighDefinition;
```

Comment:
Not from your changes, but this line appears twice, we might as well fix that now. :)

Reply:
I also got tricked by this one: it is not the same line, one is UnityEditor.* and the other UnityEngine.*
(I removed initially and then I had to revert the commit!)